website/content/docs/plugins/index.mdx

```diff
@@ -0,0 +1,200 @@
+---
+description: |
+  Packer Plugins allow new functionality to be added to Packer without modifying
+  the core source code. Packer plugins are able to add new builders,
+  provisioners, hooks, and more.
+page_title: Plugins
+sidebar_title: Packer Plugins
+---
+
+# Packer Plugins
+
+Packer Plugins allow new functionality to be added to Packer without modifying
+the core source code. Packer plugins are able to add new builders, provisioners,
+hooks, and more. In fact, much of Packer itself is implemented by writing
+plugins that are simply distributed with Packer. For example, many of the
+builders, provisioners, and more that ship with Packer are implemented as
+Plugins that are simply hardcoded to load with Packer.
```

Comment:
I would exclude this part from now on. This won't be true anymore. Maybe just make it a note that currently some plugins are embedded but they will be extracted in the following releases. 🤔 

```suggestion
hooks, and more. 
```

Reply:
I also wondered about that but I think that the `that ship with Packer` part makes it okay to keep !